kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
this function should not work, unless Normal is added to the base class. And i think this is currently not the case... (and should be discussed as a separate issue)